sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -677,6 +677,40 @@ def patch_item(
             response_hook(self.client_connection.last_response_headers, result)
         return result
 
+    @distributed_trace
+    def execute_item_batch(
+            self,
+            batch_operations: List[Tuple[Any]],
+            partition_key: Union[str, int, float, bool],
+            **kwargs) -> Dict[str, Any]:
+        """ Executes the transactional batch for the specified partition key.
+
+        :param batch_operations: The batch of operations to be executed.
+        :type batch_operations: List[Dict[str, Any]]
+        :param partition_key: The partition key value of the batch operations.
+        :type partition_key: Union[str, int, float, bool]
+        :keyword str pre_trigger_include: trigger id to be used as pre operation trigger.
+        :keyword str post_trigger_include: trigger id to be used as post operation trigger.
+        :keyword str session_token: Token for use with Session consistency.
+        :keyword str etag: An ETag value, or the wildcard character (*). Used to check if the resource
+            has changed, and act according to the condition specified by the `match_condition` parameter.
+        :keyword ~azure.core.MatchConditions match_condition: The match condition to use upon the etag.
+        :keyword Callable response_hook: A callable invoked with the response metadata.
+        :returns: A dict representing the item after the batch operations went through.
+        :raises ~azure.cosmos.exceptions.CosmosHttpResponseError: The batch did not execute successfully.
```

Comment:
This description is not accurate in the current design.

Reply:
you can still get errors thrown for issues not having to do with the transactional batch - if you send the wrong partition key, if the list of operations is empty, if the size of the request is too large, etc